src/components/Anchor/Anchor.js

```diff
@@ -208,6 +208,19 @@ class Anchor extends React.Component {
     }
   };
 
+  isActive(to, match, exact) {
+    let active;
+    if (!exact) {
+      // use the default behavior for non-exact matches
+      active = !!match;
+    } else {
+      // when we expect an exact match, dont allow react-router null
+      // value, additionally verify special chars are used
+      active = match == null ? false : match.url === to;
```

Comment:
Isn't there a `match.exact` field for this?

Reply:
`isExact`, actually https://reacttraining.com/react-router/core/api/match

Reply:
`isExact` value is incorrect as well, sample output shown below by adding a console log statement. `/frame:manageexternalnumbers` and `/frame:numberassignment` both have an `isExact` value of true even though the current url is `/frame:numberassignment`

```
[SCL] {path: "/frame:manageexternalnumbers", url: "/frame:numberassignment", isExact: true, params: {…}}
Anchor.js:180 
[SCL] {path: "/frame:numberassignment", url: "/frame:numberassignment", isExact: true, params: {…}}
```